ceph/ceph_admin/orch.py

```diff
@@ -24,6 +24,21 @@ class Orch(LSMixin, PSMixin, RemoveMixin, CephCLI):
 
     direct_calls = ["ls", "ps"]
 
+    def get_hosts_by_label(self, label: str):
+        """
+        Fetch host object by label attached to it.
+        Args:
+            label: name of the label
+        Returns:
+            hosts
+        """
+        out, _ = self.shell(args=["ceph", "orch", "host", "ls", "--format=json"])
+        hosts = list()
+        for node in loads(out):
+            if label in node.get("labels"):
+                hosts.append(node)
+        return hosts
```

Comment:
`return [node for node in loads(out) if label in node.get("labels")]`

Reply:
it would result in NameError.

Reply:
it might be pdb mode, let me see in interactive mode.